onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -398,9 +398,8 @@ def aten_elu(
 ) -> TFloat:
     """elu(Tensor self, Scalar alpha=1, Scalar scale=1, Scalar input_scale=1) -> Tensor"""
 
-    # del scale
-    # del input_scale
-    return op.Elu(self, alpha=alpha)
+    self = self * input_scale
```

Comment:
We don't need to specify op for this operation?

Reply:
Also: was this an omission-bug? (Just that the PR description didn't say this was also fixing errors.)

Reply:
We don't but updated to trace. thanks.